test_rmw_implementation/test/test_publisher.cpp

```diff
@@ -375,3 +375,27 @@ TEST_F(CLASSNAME(TestPublisherUse, RMW_IMPLEMENTATION), count_mismatched_subscri
   EXPECT_EQ(RMW_RET_OK, ret) << rmw_get_error_string().str;
   EXPECT_EQ(0u, subscription_count);
 }
+
+TEST_F(CLASSNAME(TestPublisherUse, RMW_IMPLEMENTATION), publish_message_bad_arguments) {
```

Comment:
@Lobotuerk nit: consider 
```suggestion
TEST_F(CLASSNAME(TestPublisherUse, RMW_IMPLEMENTATION), publish_message_with_bad_arguments) {
```

Reply:
Uncrustify does not seem to like that way, I'm changing it to 
```
TEST_F(
  CLASSNAME(TestPublisherUse, RMW_IMPLEMENTATION),
  publish_message_bad_arguments) {
```

Reply:
Perhaps `publish_with_bad_arguments`?

Reply:
Changed